### DISCUSSION TITLE
Error when loading Robot System

### DISCUSSION THREAD

**Discussion by ar0551** [visose/Robots#116] [Q&A] [answered]
Hi Vicente,

one of my students is encountering an issue when trying to load a new robot system (she is using the CITA UR5e), the component returns the following error:
1. Solution exception:'.', hexidezimaler Wert 0x00, ist ein ungültiges Zeichen. Zeile 1, Position 1.
Any idea what might be causing it?
Thanks,
Andrea

**Comment by visose** [comment, marked as answer]
Hi, they probably downloaded the html GitHub page of the xml rather than the actual XML. There's a button that says 'raw' you can use to download it.
